app/static/src/app/utils.ts

```diff
@@ -31,6 +31,15 @@ export interface ProcessFitDataResult {
     timeVariableCandidates: string[] | null,
     error?: Error
 }
+
+export function getCodeErrorFromResponse(errorResponse: OdinModelResponseError): Error {
```

Comment:
Should probably add a test for this in utils.test.ts

Reply:
Tests were added, probably in a wrong place. It's been moved to utils.test now.